# Genetic mutation rate
The genetic mutation rate (Ka_Sc v.s. Kl) obtained from "The Hsp90-Dependent Proteome Is Conserved and Enriched for Hub Proteins with High Levels of Proteinâ€“Protein Connectivity" (https://academic.oup.com/gbe/article/6/10/2851/614233). The top 20 result provide in `example/output/Ka_Sc_vs_Kl/cor_sig_local.csv`. These candidate may alter the regulation of genome stability thus correlated with the genetic mutation rate phenotype.

# Expression noise
The two expression noise is obtained from "Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise" (https://www.nature.com/articles/nature04785). The dataset contain two folder `Noise_SD` and `Noise_YEPD`, which means the expression noise is measure in two different medium. The top 20 result provide in `example/output/Noise_SD/cor_sig_local.csv` and `example/output/Noise_YEPD/cor_sig_local.csv`. The candidate of these list may involved in increase or reduce the expression noise and thus correlated with the expression noise meausement. The user may use these candidate and perform knockout experiment and meause the expression noise again to check the expression noise level change.

# Protein-protein interactions (Physical interaction)
The physcial protein-protein (PPI) interaction obtained from "BioGRID: a general repository for interaction datasets" (https://academic.oup.com/nar/article/34/suppl_1/D535/1133554). The dataset contain the physical protein-protein interaction number of a each gene. The top 20 result provide in `example/output/Physical_interaction/cor_sig_local.csv`. The candidate may involved in control the PPI domain stability thus maintain the physical PPI number. The user may use the knockout strain of these candidate to performance experimental evolution experiments and measure the PPI number changed after experiment.

# Transcriptional responsivness
The transcriptional responsivness obtained from "A genetic signature of interspecies variations in gene expression (https://www.nature.com/articles/ng1819). These data is the measurement of expression variation from over 1500 conditions. The top 20 result provide in `example/output/Transcription_responsiveness/cor_sig_local.csv`. The candidate may control the expression variability unver different condition. The user can use the deletion strain of these candidate and measure gene expression variability under serval condition to verify the gene function.

# Recombination rate
The recombinate rate data obtained from "High-resolution mapping of meiotic crossovers and non-crossovers in yeast" (https://www.nature.com/articles/nature07135). The data represent the recombination rate of each gene during meiosis. The top 20 result provide in `example/output/Recombination_rate/cor_sig_local.csv`. The candidate may influence the recombinase efficiency during crossover in meiosis. The user may design an *in vivo* experiment to measure and verify the recombinase activity change.