# Genetic mutation rate
The genetic mutation rate (Ka_Sc v.s. Kl) obtained from "The Hsp90-Dependent Proteome Is Conserved and Enriched for Hub Proteins with High Levels of Protein-Protein Connectivity" (https://academic.oup.com/gbe/article/6/10/2851/614233). The top 20 results provide in `example/output/Ka_Sc_vs_Kl/cor_sig_local.csv`. These candidates may alter the regulation of genome stability thus correlated with the genetic mutation rate phenotype.

# Expression noise
The two expression noise is obtained from "Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise" (https://www.nature.com/articles/nature04785). The dataset contains two folders `Noise_SD` and `Noise_YEPD`, which means the expression noise is measure in two different mediums. The top 20 results provide in `example/output/Noise_SD/cor_sig_local.csv` and `example/output/Noise_YEPD/cor_sig_local.csv`. The candidate of these lists may be involved in the increase or reduce the expression noise and thus correlated with the expression noise measurement. The user may use these candidates and perform knockout experiments and measure the expression noise again to check the expression noise level change.

# Protein-protein interactions (Physical interaction)
The physical protein-protein (PPI) interaction obtained from "BioGRID: a general repository for interaction datasets" (https://academic.oup.com/nar/article/34/suppl_1/D535/1133554). The dataset contains the physical protein-protein interaction number of each gene. The top 20 results provide in `example/output/Physical_interaction/cor_sig_local.csv`. The candidate may be involved in control the PPI domain stability thus maintain the physical PPI number. The user may use the knockout strain of these candidates to perform experimental evolution experiments and measure the PPI number changed after the experiment.

# Transcriptional responsiveness
The transcriptional responsiveness obtained from "A genetic signature of interspecies variations in gene expression (https://www.nature.com/articles/ng1819). This data is the measurement of expression variation from over 1500 conditions. The top 20 results provide in `example/output/Transcription_responsiveness/cor_sig_local.csv`. The candidate may control the expression variability under different conditions. The user can use the deletion strain of these candidates and measure gene expression variability under serval conditions to verify the gene function.

# Recombination rate
The recombination rate data obtained from "High-resolution mapping of meiotic crossovers and non-crossovers in yeast" (https://www.nature.com/articles/nature07135). The data represent the recombination rate of each gene during meiosis. The top 20 results provide in `example/output/Recombination_rate/cor_sig_local.csv`. The candidate may influence the recombinase efficiency during crossover in meiosis. The user may design an *in vivo* experiment to measure and verify the recombinase activity change.
